en/core-libraries/time.rst

```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
 It is recommended to always use the constants instead of directly passing a date
 format string.
 
+.. versionadded:: 3.2.11
+
+You can change the default timezone used for outputting your whole app in `app.php`:
+
+    'defaultOutputTimezone' => 'Europe/Berlin',
+
+... or overwrite fetching from app config in `bootstrap.php`:
+
+    Time::setDefaultOutputTimezone('Europe/Berlin');
+    FrozenTime::setDefaultOutputTimezone('Europe/Berlin');
+
+At any point in your application, say in a `FrontendAppController` you may change the
+default output timzeone by calling `setDefaultOutputTimezone()`, for example:
+
+    use Cake\I18n\FrozenTime;
+    use Cake\I18n\Time;
+
+    // ...
+
+    public function initialize() {
+        parent::initialize();
+        Time::setDefaultOutputTimezone('America/Vancouver');
```

Comment:
You could save the Timezone string into a variable and use that one, to better showcase using it for both.